src/engine/api/engine-api.ts

```diff
@@ -80,6 +84,12 @@ export class IvyEngineApi {
     await this.runPostRequest(newProjectParams, CREATE_PROJECT_REQUEST);
   }
 
+  public async deleteProject(projectDir: string): Promise<any> {
+    const params = { projectDir };
+    const url = this.projectRequestURL(DELETE_PROJECT_REQUEST.sourcePath).toString();
+    return new Promise(resolve => axios.delete(url, { params, headers, auth }).then(response => resolve(response.data)));
```

Comment:
why is this wrapping in a new promise needed?
something like the following should also work I think, or not?
`axios.delete(url, { params, headers, auth }).then(response => response.data)`

Reply:
thanks, you are right, this promise is useless